time/src/main/java/net/jqwik/time/internal/properties/arbitraries/DefaultOffsetTimeArbitrary.java

```diff
@@ -0,0 +1,205 @@
+package net.jqwik.time.internal.properties.arbitraries;
+
+import java.time.*;
+import java.time.temporal.*;
+
+import org.apiguardian.api.*;
+
+import net.jqwik.api.*;
+import net.jqwik.api.arbitraries.*;
+import net.jqwik.time.api.*;
+import net.jqwik.time.api.arbitraries.*;
+
+import static java.time.temporal.ChronoUnit.*;
+import static org.apiguardian.api.API.Status.*;
+
+@API(status = INTERNAL)
+public class DefaultOffsetTimeArbitrary extends ArbitraryDecorator<OffsetTime> implements OffsetTimeArbitrary {
+
+	private OffsetTime offsetTimeMin = OffsetTime.MIN;
+	private OffsetTime offsetTimeMax = OffsetTime.MAX;
+
+	private LocalTime timeMin = null;
```

Comment:
Why don't you just delegate all the constraint values timeMinMax etc. to an embedded LocalTimeArbitrary?